code/src/App.js

```diff
@@ -1,9 +1,25 @@
 import React from 'react'
+import { BrowserRouter, Switch, Route } from 'react-router-dom'
+
+import { MovieList } from './pages/MovieList'
+import { MovieDetails } from './pages/MovieDetails'
+import { NotFound } from './pages/NotFound'
 
 export const App = () => {
+
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+  <BrowserRouter>
+    <Switch>
+      <Route path ="/" exact>
+        <MovieList />
+      </Route>
+      <Route path="/movies/:movieId" exact>
+        < MovieDetails/>
+      </Route>
+      < Route>
+        <NotFound />
```

Comment:
Really nice thought of implementing this component but we can't really see how it's used in the project? We do however see your comment in readme and that you might have had an issue with this. 
Nevertheless good job! :)